.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: Build macOS, Ubuntu, Windows
+
+on:
+  pull_request:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+*'
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release ${{ matrix.target }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            profile: release+static
+            target: x86_64-unknown-linux-musl
+            needs_musl: true
+
+          - os: ubuntu-latest
+            profile: release+static+aarch64
+            target: aarch64-unknown-linux-musl
+            needs_musl: true
+
+          - os: macos-11
+            profile: release+mac
+            target: x86_64-apple-darwin
+
+          - os: macos-11
+            profile: release+mac+aarch64
+            target: aarch64-apple-darwin
+
+            - os: windows-2022
+            profile: release+windows
+            target: x86_64-pc-windows-msvc
+            suffix: .exe
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Setup cargo make
+        uses: davidB/rust-cargo-make@v1
+
+      - name: Store git tag vars
+        shell: bash
+        run: echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
```

Comment:
we dont actually need this / use this anywhere

(we could however, merge `publish.yml` into this `ci.yml` so we won't have to build them twice, i.e. 
https://github.com/swift-nav/libsbp/blob/6cbf963cb708928664d21761c5bf7d2d095ffe8f/.github/workflows/rust.yaml#L100
just a `if: startsWith(github.ref, 'refs/tags/v')` check if its a tag for release stage etc... which in that case will need to run this on all `push:` )

Reply:
Removed.

Paraphrasing our conversation of the (parenthesis) part for @silverjam :
merge could add complications, would rather make it a minor followup ticket